config/crd/bases/infrastructure.cluster.x-k8s.io_azureclusteridentities.yaml

```diff
@@ -56,11 +56,11 @@ spec:
                   either a Service Principal password or certificate secret.
                 properties:
                   name:
-                    description: Name is unique within a namespace to reference a
+                    description: name is unique within a namespace to reference a
```

Comment:
if you're curious, this is what changed https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/core/v1/types.go#L929

the diff is generated by the k8s.io/api bump